network/static/js/observation_view.js

```diff
@@ -81,5 +81,17 @@ $(document).ready(function() {
         });
     });
 
+    // Delete confirmation
```

Comment:
It's not, but it's not new code either. This was inline javascript on the template and I just moved it to the script file. I'll add a comment on the commit.